extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +140,486 @@ For example
+
+> **Implementation note:** Use of floating-point properties in a property texture would require a floating point-compatible image format like KTX2 provided by an additional extension.
+
+> **Example:** Property texture implementing a "thermalSample" class, with property values stored in a glTF texture at index 0 and indexed by `TEXCOORD_0`.
+>
+> <img src="figures/feature-texture.png"  alt="Property Texture" width="500">
+>
+> _Class and property texture_
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_mesh_features": {
+>       "schema": {
+>         "classes": {
+>           "wall": {
+>             "name": "Wall Temperature vs. Insulation",
+>             "properties": {
+>               "insideTemp": {
+>                 "name": "Inside Temperature",
+>                 "type": "UINT8"
+>               },
+>               "outsideTemp": {
+>                 "name": "Outside Temperature",
+>                 "type": "UINT8"
+>               },
+>               "insulation": {
+>                 "name": "Insulation Thickness",
+>                 "type": "UINT8",
+>                 "normalized": true
+>               },
+>             }
+>           }
+>         }
+>       },
+>       "propertyTextures": [{
+>         "class": "wall",
+>         "index": 0,
+>         "texCoord": 0,
+>         "properties": {
+>           "insideTemp": [0],
+>           "outsideTemp": [1],
+>           "insulation": [2]
+>         }
+>       }]
+>     }
+>   }
+> }
+> ```
+>
+> _Primitive_
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "attributes": {
+>         "POSITION": 0,
+>         "TEXCOORD_0": 1
+>       },
+>       "indices": 2,
+>       "material": 0,
+>       "extensions": {
+>         "EXT_mesh_features": {
+>           "propertyTextures": [0]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+
+A `propertyTexture` object extends the glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `texCoord` specifies a texture coordinate set in the referring primitive.
+
+The `properties` map specifies the texture channels providing data for available properties. An array of integer index values identifies channels, where multiple channels may be used only for fixed-length arrays of 2, 3, or 4 components. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. All values are stored in linear space.
+
+> **Example:** A property texture specifying that "insideTemp" property values are stored in the red channel, and "insulation" property values are stored in the green channel, both indexed by UV coordinates in a `TEXCOORD_0` attribute.
+>
+> ```jsonc
+> // Root EXT_mesh_features extension:
+> {
+>   "propertyTextures": [{
+>     "class": "wall",
+>     "index": 0,
+>     "texCoord": 0,
+>     "properties": {
+>       "insideTemp": [0],
+>       "insulation": [1]
+>     }
+>   }]
+> }
```

Comment:
Hm, maybe for this paragraph we should have a different example that has a vec2 property so we can show what it looks like if you need to select multiple components. If so, the paragraph above would need to be updated too.

e.g.

```
{
  "propertyTextures": [{
    "class": "river",
    "index": 0,
    "texCoord": 0,
    "properties": {
      "velocity": [0, 1],
      "depth": [2]
    } 
  }]
}
```

Reply:
Agreed – good addition, and this helps to cover the remaining details that were in the Core Metadata spec's "raster format" section but not here yet. I've used wind speed and direction as the example.